clang/test/Driver/sycl-int-footer.cpp

```diff
@@ -24,9 +24,15 @@
 
 /// Check that integration footer can be disabled
 // RUN:  %clangxx -fsycl -fno-sycl-use-footer %s -### 2>&1 \
-// RUN:   | FileCheck -check-prefix NO-FOOTER --implicit-check-not "-fsycl-int-footer" %s
+// RUN:   | FileCheck -check-prefix NO-FOOTER --implicit-check-not "-fsycl-int-footer" --implicit-check-not "-fsycl-use-main-file-name" %s
 // NO-FOOTER: clang{{.*}} "-fsycl-is-device"{{.*}} "-fsycl-int-header=[[INTHEADER:.+\.h]]" "-sycl-std={{.*}}"
-// NO-FOOTER: clang{{.*}} "-include" "[[INTHEADER]]"{{.*}} "-fsycl-is-host"{{.*}} "-o"
+// NO-FOOTER: clang{{.*}} "-include" "[[INTHEADER]]"{{.*}} "-fsycl-is-host"{{.*}} "-main-file-name" "sycl-int-footer.cpp"{{.*}} "-o"
+
+// Test that -fsycl-use-main-file-name is not passed if -fsycl is not passed.
+// This is test is located here, because -fsycl-use-main-file-name is tightly
```

Comment:
Did you mean to use "// This test is located here, instead // This is test is located here,?